packages/css-library/tokens/breakpoints.json

```diff
@@ -1,7 +1,19 @@
 {
+  "xsmall-screen": {
+    "value": "320px"
+  },
   "small-screen": {
     "value": "481px"
   },
+  "medium-screen": {
+    "value": "640px"
+  },
+  "small-desktop-screen": {
+    "value": "1008px"
+  },
+  "medium-large-screen": {
+    "value": "640px"
+  },
```

Comment:
Formation has a strange [duplicate token](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/base/_b-breakpoints.scss#L44-L45) that gives the same value as `$medium-screen`:

```
$medium-large-screen: 768px;
$medium-screen:       $medium-large-screen;
```

Both of these tokens are being used in both Formation and vets-website. Our documentation only lists these six tokens for breakpoints though:

![Screenshot 2024-01-30 at 12 19 03 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/171a4aa1-5e72-456a-8831-c2f0a6b8c8be)

Instead of adding `medium-large-screen` to the CSS Library, can we instead replace all places that it's being used with  `medium-screen`? Then we won't have to continue supporting that extra duplicate token.

Reply:
A vets-website PR for replacing this token is here: https://github.com/department-of-veterans-affairs/vets-website/pull/27757

Reply:
Sounds like a good plan to me!